spring-cloud-circuitbreaker-spring-retry/src/main/java/org/springframework/cloud/circuitbreaker/springretry/SpringRetryAutoConfiguration.java

```diff
@@ -36,14 +35,15 @@
 @ConditionalOnClass(RetryTemplate.class)
 public class SpringRetryAutoConfiguration {
 
-	@Autowired(required = false)
```

Comment:
just to be on par with the general recommendation (if you are OK with this, of course), to drop field injection.